core/src/main/resources/lib/layout/breadcrumbs.js

```diff
@@ -83,10 +83,14 @@ window.breadcrumbs = (function () {
   }
 
   function postRequest(action, event, url) {
-    new Ajax.Request(url);
-    if (event.length == 1 && event[0].target != null) {
-      hoverNotification("Done.", event[0].target);
-    }
+    fetch(url, {
+      method: "post",
+      headers: crumb.wrap({}),
+    }).then(() => {
+      if (event.length == 1 && event[0].target != null) {
+        hoverNotification("Done.", event[0].target);
+      }
+    });
```

Comment:
Is this really equivalent to the old code? My understanding of the old code is that it would fire off the request and then execute the `if` block without waiting for the response. Your new version waits for the response before executing the `if` block. Was that an intentional change in behavior because you think the old behavior was wrong? My general disposition is that we should minimize changes in behavior to minimize the risk of regression, but I could be persuaded otherwise.

Reply:
I think an improvement but I've reverted it to minimise change.